guides/source/configuring.md

```diff
@@ -1027,6 +1027,8 @@ You can find more detailed configuration options in the
 
 * `config.active_storage.previewers` accepts an array of classes indicating the image previewers available in Active Storage blobs. The default is `[ActiveStorage::Previewer::PopplerPDFPreviewer, ActiveStorage::Previewer::MuPDFPreviewer, ActiveStorage::Previewer::VideoPreviewer]`. `PopplerPDFPreviewer` and `MuPDFPreviewer` can generate a thumbnail from the first page of a PDF blob; `VideoPreviewer` from the relevant frame of a video blob.
 
+* `config.active_storage.optimizers` accepts an array of classes indicating the image optimizers available for Active Storage blobs. The default is `[ActiveStorage::Optimizer::WebImageOptimizer]`. The web image optimizer will not apply any optimizations to web images, but will convert variable image blobs that are not web images into PNGs.  
```

Comment:
If users remove `WebImageOptimizer` will this mean they'd break their variants/representations for e.g. pdfs?

Reply:
No. Removing `WebImageOptimizer` would remove active storage's ability to convert images that are variable but are not considered web images (such as tiff and bmp) into PNGs, but that's it. 

Not sure why someone would want to do that, but if necessary we can protect against it.